pipelines/nlu-extension-build.yml

```diff
@@ -0,0 +1,74 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - /extensions/
+    - /pipelines/extension-build.yml
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - /extensions/
+    - /pipelines/extension-build.yml
+
+stages:
+- stage: Build_private
+  jobs:
+  - job: build_vsix
+    pool:
+      vmImage: 'ubuntu-latest' 
+    steps:
+    - task: NPM@1
+      displayName: npm install
+      inputs:
+        command: install
+        workingDir: extensions
+
+    - task: NPM@1
+      displayName: npm run lint
+      inputs:
+        command: custom
+        customCommand: run lint
+        workingDir: extensions
+
+    - task: NPM@1
+      displayName: npm run build
+      inputs:
+        command: custom
+        customCommand: run build
+        workingDir: extensions
+
+    - task: CopyFiles@2
+      displayName: Stage extension artifact for publish
+      inputs:
+        sourceFolder: extensions
+        contents: '**/*.vsix'
+        targetFolder: $(Build.ArtifactStagingDirectory)
+
+    - task: PublishPipelineArtifact@0
+      displayName: Publish extension artifact
+      inputs:
+        artifactName: drop
+        targetPath: $(Build.ArtifactStagingDirectory)
+
+    - task: TfxInstaller@2
+      displayName: 'Install Node CLI'
```

Comment:
Remove single quotes where not needed. The only place I found that actually required them was for the `contents` input (i.e., `**.*.vsix`)

Reply:
done